lib/linter/node-event-generator.js

```diff
@@ -166,23 +194,32 @@ function tryParseSelector(rawSelector) {
     }
 }
 
+const selectorCache = {};
```

Comment:
Can we use `Map` instead of object here? Selectors can be user-defined (e.g., in `no-restricted-syntax` rule), so it's better to avoid usual issues with object-maps.

Reply:
Resolved in a1bd47baa.